src/context/auth-context.jsx

```diff
@@ -4,34 +4,43 @@ import * as auth from "../service/auth-services";
 
 const AuthContext = createContext();
 
-
 function AuthProvider(props) {
   const [user, setUser] = useState(null);
+  const [recruiter, setRecruiter] = useState(null);
+
+  // useEffect(() => {
+  //   getUser().then(setUser).catch(console.log);
+  // }, []);
 
-  useEffect(() => {
-    getUser().then(setUser).catch(console.log);
-  }, []);
-  
   function login(credentials) {
     auth.login(credentials).then(setUser).catch(console.log);
   }
 
+  function lgoinRecruiter(credentials) {
+    auth.loginRecruiter(credentials).then(setRecruiter).catch(console.log);
+  }
```

Comment:
I caught a little typo

```suggestion
  function loginRecruiter(credentials) {
    auth.loginRecruiter(credentials).then(setRecruiter).catch(console.log);
  }
```